src/app/component/layout/components/search-all-results/search-all-results.component.ts

```diff
@@ -48,13 +48,21 @@ export class SearchAllResultsComponent implements OnInit, OnDestroy {
 
   private getSearchData(data: SearchModel): void {
     this.getNews(data.ecoNews);
-    // tslint:disable-next-line:no-unused-expression
-    data.countOfResults ? this.itemsFound = data.countOfResults : null;
+    if (data.countOfResults) {
+      this.itemsFound = data.countOfResults;
+    } else {
+      this.itemsFound = null;
+    }
     this.spliceResults();
   }
 
   private getNews(news): void {
-    (news && news.length > 0) ? (this.isSearchFound = true, this.elements = news) : (this.isSearchFound = false);
+    if (news && news.length > 0) {
+      this.isSearchFound = true;
+      this.elements = news;
+    } else {
+      this.isSearchFound = false;
+    }
```

Comment:
the same as previous. But a slightly different, I'd propose:
```js
this.isSearchFound = news && news.length;
this.elements = this.isSearchFound ? news : this.elements;
```
- looks more simplify as both

Reply:
Thanks, fixed